scripts/setup-amd.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+if [ $# -eq 0 ]; then
+    ROCM_VERSION="5.7.1"
+else
+    ROCM_VERSION=$1
+fi
+
+ml load rocm/${ROCM_VERSION}
+
+LLVM_INSTALL_DIR=${ROCM_PATH}/llvm
+
+mkdir build-rocm-${ROCM_VERSION}
```

Comment:
I don't like the file creation path. Add a $(hostname) field or ${SYS_TYPE} if this is only for internal systems. When we add additional clusters this can be an issue.

Reply:
I'll defer that, it's up to the user. We don't use the script for CI automation.